tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +693,649 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+                                  headers={'Content-type': 'application/json'})
+        # print(response)
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['backup_links'][0]['endpoint_a'] == payload['backup_links'][0]['endpoint_a']
+        assert data['backup_links'][0]['endpoint_b'] == payload['backup_links'][0]['endpoint_b']
+
+    """It is returning Response 500, should be 200"""
+    @pytest.mark.xfail
+    def test_150_patch_primary_links(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ],
+            "primary_links": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}],
+        }
+        response = requests.post(api_url, data=json.dumps(payload),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        time.sleep(10)
+
+        payload = {
+            "primary_links": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ]
+        }
+
+        # It sets a new circuit's backup_links
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        # print(response)
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['primary_links'][0]['endpoint_a'] == payload['primary_links'][0]['endpoint_a']
+        assert data['primary_links'][0]['endpoint_b'] == payload['primary_links'][0]['endpoint_b']
+
+    def test_155_patch_on_dynamic_backup_path_and_primary_path(self):
```

Comment:
actually, I didn't see the patch here

Reply:
Resolved with commit [748a6ad](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/748a6ad741f70941e4528b147cc4c844214d6e95)